### DISCUSSION TITLE
Given a config file, is there a way to tell if a rule within actually exists?

### DISCUSSION THREAD

**Discussion by Nfinished** [eslint/eslint#16519] [Q&A] [answered]
I'm writing tests for an eslint config file and after scouring the docs I haven't found a way to identify nonexistent rules that might have been accidentally specified. Is this possible to do?

I'm assuming this must exist at some level because the vscode extension seems to be aware of missing rules.
![Image 2022-11-07 at 11 39 35 AM](https://user-images.githubusercontent.com/1719791/200365361-268c2255-715c-4062-9d29-14f708048f6b.jpg)

**Comment by mjmcgrath2010** [comment]
+1

**Comment by Nfinished** [comment, marked as answer]
As I pointed out to myself in the question without realizing it, this is of course an error and can be discovered by linting a file. Whoops.
